app/assets/stylesheets/application.scss

```diff
@@ -34,3 +34,23 @@ dl.govuk-\!-margin-bottom-0 > .govuk-summary-list__row > .govuk-summary-list__ke
     border-bottom: 1px solid $govuk-border-colour;
   }
 }
+
+.govuk-table__cell {
```

Comment:
This has messed with the payment history view as it also uses a govuk table. Can you use a custom class for these changes to ensure this page the only thing being affected